contracts/rust/src/cape/faucet.rs

```diff
@@ -34,26 +34,34 @@ async fn test_faucet() -> Result<()> {
     assert!(!contract.faucet_initialized().call().await?);
     assert_eq!(contract.deployer().call().await?, deployer.address());
 
+    let pub_key_bytes = bincode::serialize(&faucet_manager.pub_key()).unwrap();
```

Comment:
I think this is fine because it's just a test and the contract itself doesn't care what the pub key bytes are, but in general we should always be consistent (either always use `TaggedBase64` or always use `bincode` bytes) so that consumers of the event (e.g. EQS) know what format to expect